model-server/src/main/kotlin/org/modelix/model/server/Main.kt

```diff
@@ -219,11 +227,21 @@ object Main {
                                     li {
                                         a("user") { +"View JWT token and permissions" }
                                     }
+                                    li {
+                                        a("swagger") { +"SwaggerUI" }
+                                    }
                                 }
                             }
                         }
                         call.respondText("Model Server")
                     }
+                    if (cmdLineArgs.noSwaggerUi) {
+                        get("swagger") {
+                            call.respondText("SwaggerUI is disabled")
+                        }
+                    } else {
+                        swaggerUI(path = "swagger", swaggerFile = "../api/model-server.yaml")
```

Comment:
How does this work with the other definitions?

Reply:
we only expose the public one

Reply:
```suggestion
                        // we only serve the public API to the outside via swagger UI
                        swaggerUI(path = "swagger", swaggerFile = "../api/model-server.yaml")
```